polkadot/primitives/src/vstaging/mod.rs

```diff
@@ -50,6 +51,52 @@ impl Default for ApprovalVotingParams {
 	}
 }
 
+/// Coretime configuration parameters
+#[derive(
+	RuntimeDebug,
+	Copy,
+	Clone,
+	PartialEq,
+	Encode,
+	Decode,
+	TypeInfo,
+	serde::Serialize,
+	serde::Deserialize,
+)]
+pub struct CoretimeParams<BlockNumber> {
+	/// How many cores are managed by the coretime chain.
+	pub cores: u32,
+	/// The max number of times a claim can time out in availability
+	pub max_availability_timeouts: u32,
+	/// The number of blocks a claim stays in the scheduler's claimqueue before getting cleared.
+	/// This number should go reasonably higher than the number of blocks in the async backing
+	/// lookahead.
+	pub ttl: BlockNumber,
```

Comment:
Maybe we should put the lookahead in the same struct and maybe even consider enforcing that relationship.

Putting these into their own struct seems ok, but I would rename it to `SchedulerParams` and add remaining scheduler variables, like the lookahead as well.

Reply:
I didn't see anything scheduler related besides coretime/ondemand params and `scheduling_lookahead`. Am I missing something?

Reply:
Done